### DISCUSSION TITLE
Notes for Beta Projects

### DISCUSSION THREAD

**Discussion by chrisgaun** [community/community#5795] [Projects and Issues]
The Beta Project Boards no longer have "notes" or allow us to add topics to status columns  

We used notes in the GA Projects  to convey information about the definition of done for each stage of column status (examples attached). This allowed us to all be on same page and quickly onboard new members. 

The new Beta boards no longer have "notes
![Screen Shot 2021-09-24 at 2 17 09 PM](https://user-images.githubusercontent.com/10170069/134722401-1a8a2651-1767-48f5-8886-5cac01c2a978.png)
"
![Screen Shot 2021-09-24 at 2 17 15 PM](https://user-images.githubusercontent.com/10170069/134722407-b41874fb-22a2-4fcb-aa1d-d2e8ec053886.png)

**Comment by lnryan** [comment]
We make heavy use of notes with checklists for ephemeral or non-technical project work that doesn't warrant all the functionality of issues and that frankly would only pollute the issue archive. We also use notes to collect related issues since there's no concept of sub-tasks in github. This is a struggle for team members coming from Jira and trying to adopt Github instead. Not having these features would likely result in us being forced to return to Jira.

**Comment by guylepage3** [comment]
+1 for heavy use of notes. Thank you

**Comment by guylepage3** [comment]
Would also be nice to increase the character limit of notes to 1500 characters as well. Thanks.
